docs/source/what_is_xdem.rst

```diff
@@ -0,0 +1,16 @@
+.. _what_is_xdem:
+
+What is xDEM?
+=============
+
+*xdem* aims at simplifying the analysis of Digital Elevation Models (DEM). This module provides typical post-processing steps such as coregistration, elevation/volume change measurements and error statistics.
```

Comment:
Right now this line is a copy/paste of the `index.rst` first line. If a reader reads the introduction page, then goes to "What is xDEM" which is the first link below, he re-reads this same line twice in a row.
We could leave this shortened version only on the landing page, and elaborate in this one?